packages/client/hmi-client/src/page/project/components/code-editor.vue

```diff
@@ -30,16 +30,27 @@ import FileUpload from 'primevue/fileupload';
 import Button from 'primevue/button';
 import '@node_modules/ace-builds/src-noconflict/mode-python';
 import '@node_modules/ace-builds/src-noconflict/theme-chrome';
-import { ref } from 'vue';
+import { onMounted, ref } from 'vue';
 import { logger } from '@/utils/logger';
 import { VAceEditorInstance } from 'vue3-ace-editor/types';
 import API from '@/api/api';
+import { isEmpty } from 'lodash';
 
 const DEFAULT_TEXT = '# Paste some python code here or import from the controls above';
-const content = ref(DEFAULT_TEXT);
+const content = ref<string>(DEFAULT_TEXT);
 const editor = ref<VAceEditorInstance['_editor'] | null>(null);
 const selectedText = ref('');
 
+const props = defineProps<{
+	code?: string;
+}>();
```

Comment:
Personally I find optional properties annoying to deal with, you end up sometimes doing extra checks that are seeming unnecessary just to satisfy the linter.

How do we feel about making `code` required and default to empty string instead?

Reply:
I disagree.
A simple test if better, they can be null and that's it.